html-forms/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+
+  <form>
+    <div><label for="email">Email</label></div>
+    <input type="text" name="email" id="email"/>
+
+    <div><label for="why is coding cool">Why is coding cool?</label></div>
```

Comment:
`for` attribute values and `id` attribute values should not include spaces, please change this to kebab-case